cmd/run.go

```diff
@@ -149,6 +150,7 @@ func runRun(ctx context.Context, ro options.RunOptions, args []string, signers .
 		witness.RunWithTimestampers(timestampers...),
 	)
 	if err != nil {
+		logAttestorFailure(attestors, ro.WorkingDir, err)
```

Comment:
I think we do need a way to log here but the mapping to extends logs with more meaningful content might not be the right way. Probably we need a better Exception object to pass more context and use that. But I don't know which framework might best fulfill that gap.